generators/app/templates/readme.md

```diff
@@ -1,7 +1,87 @@
-# <%= projectname %>
+# UI5 Application <%= namespace %>
 
-Insert the purpose of this project and some interesting infos here
+Insert the purpose of this project and some interesting info here...
 
-## Credits
+## Description
 
-This project has been generated with 💙 and [easy-ui5](https://github.com/SAP)
+This app demonstrates a setup for developing UI5 applications.
```

Comment:
I don't think this fits for FLP plugins, does it?

Reply:
changed it to FLP Plugin